holoviews/plotting/bokeh/element.py

```diff
@@ -1111,10 +1111,16 @@ def _update_main_ranges(self, element, x_range, y_range, ranges):
             self._update_range(x_range, l, r, xfactors, self.invert_xaxis,
                                self._shared['x-main-range'], self.logx, streaming)
         if not self.drawn or yupdate:
-            self._update_range(y_range, b, t, yfactors,
-                               y_range.tags[1]['invert_yaxis'] if y_range.tags else False,
-                               self._shared['y-main-range'], self.logy, streaming)
+            self._update_range(
+                y_range, b, t, yfactors, self._get_tag(y_range, 'invert_yaxis'),
+                self._shared['y-main-range'], self.logy, streaming
+            )
 
+    def _get_tag(self, element, tag_name):
```

Comment:
Not sure what contains tags, so I chose the generic name of `element`. 

If only a limited set of Bokeh models have tags, I would like to update the element and maybe also the method name.